frictionless/program/validate.py

```diff
@@ -263,7 +262,42 @@ def program_validate(
                         error.message,
                     ]
                 )
+                # error list for summary
+                error_title = f"{error.name} ({error.code})"
+                if error_title not in error_list:
+                    error_list[error_title] = 0
+                error_list[error_title] += 1
+                if task.partial:
+                    last_row_checked = error.get("rowPosition", "")
             content = _wrap_text_to_colwidths(content)
+        # summary
+        rows_checked = last_row_checked if task.partial else None
+        summary_content = helpers.validation_summary(
+            source,
+            basepath=task.resource.basepath,
+            timer=timer,
+            rows_checked=rows_checked,
+            error_list=error_list,
+        )
+        typer.echo("\n# Summary \n")
+        if task.partial:
+            typer.echo(
+                "The document was partially validated because of one of the limits"
+            )
+            typer.echo("* limit errors")
+            typer.echo("* memomry Limit \n")
```

Comment:
memomry

Reply:
I have missed this change. I will fix it in next PR.